applications/openshift/networking/routes_rate_limit/rule.yml

```diff
@@ -0,0 +1,49 @@
+prodtype: ocp4
```

Comment:
lets add this rule here: https://github.com/ComplianceAsCode/content/blob/master/controls/nist_ocp4.yml